prow/cmd/deck/static/script.js

```diff
@@ -1,15 +1,16 @@
 "use strict";
 
 function getParameterByName(name) {  // http://stackoverflow.com/a/5158301/3694
-    var match = RegExp('[?&]' + name + '=([^&/]*)').exec(window.location.search);
+    var match = RegExp('[?&]' + name + '=([^&/]*)').exec(
```

Comment:
I'd rather not get into the habit of checking in diffs like this that are simply from a formatter

Reply:
how do we ignore the diff from formatter?

Reply:
Don't make it in the place :)

Reply:
if it's not super trivial to un-do these diffs, we can leave them

Reply:
Thanks for pointing out! I will be more careful to use the formatter